class.jetpack.php

```diff
@@ -1452,6 +1452,13 @@ public static function refresh_active_plan_from_wpcom() {
 	 * @return array Active Jetpack plan details
 	 */
 	public static function get_active_plan() {
+		global $active_plan_cache;
```

Comment:
Is there any better way to do this than introducing another global?

Reply:
Unfortunately I couldn't think of one.

Reply:
To be clear: We could refactor the whole way that we list out modules, and then we'd only need to call this one - but that requires a lot more time than "hack week" necessarily allows.